src/rebar_prv_compile.erl

```diff
@@ -103,3 +107,23 @@ handle_args(State) ->
     {Args, _} = rebar_state:command_parsed_args(State),
     Jobs = proplists:get_value(jobs, Args, ?DEFAULT_JOBS),
     {ok, rebar_state:set(State, jobs, Jobs)}.
+
+copy_app_dirs(AppName, OldAppDir, AppDir) ->
+    case ec_cnv:to_binary(filename:absname(OldAppDir)) =/=
+        ec_cnv:to_binary(filename:absname(AppDir)) of
+        true ->
+            Name = ec_cnv:to_list(AppName),
+            AppFile = filename:join([OldAppDir, "ebin", Name++".app"]),
+            case filelib:is_file(AppFile) of
+                true ->
+                    file:copy(AppFile,
+                              filename:join([AppDir, "ebin", Name++".app"]));
+                false ->
+                    ok
+            end,
+            filelib:ensure_dir(filename:join(AppDir, "dummy")),
+            [file:make_symlink(filename:join(OldAppDir, Dir), filename:join(AppDir, Dir))
+            || Dir <- ["priv", "include"]];
```

Comment:
Na, can't do copy. People use `priv` for lots of stuff, like web pages, that would be best to be able to have change without running `rebar3` again to copy to the build output. And the fact that it could have large files and that we'd need to delete files that no longer exist.

In the compiler we'll just have to check that `priv` and `include` exist and aren't deadlinks and make them in the appdir (not the output dir) if they don't.